x-pack/plugins/security_solution/public/common/components/sourcerer/misc.test.tsx

```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I'm not sure what the correct naming for this file would be, but I think the sourcerer code and tests could be broken apart / modularized better. I'll open a separate issue for it

Reply:
@michaelolo24 create the issue https://github.com/elastic/kibana/issues/171271